apps/dashboard/app/views/batch_connect/session_contexts/_path_selector.html.erb

```diff
@@ -39,20 +39,23 @@
           <span aria-hidden="true">&times;</span>
         </button>
       </div>
-      <div class="modal-body">        
+      <div class="modal-body">       
         <div>
-            <ol id="<%= breadcrumb_id %>" class="breadcrumb breadcrumb-no-delimiter">
-            </ol>
-            <table class="table table-hover table-condensed d-table w-100" id="<%= table_id %>">
-                <thead>
-                  <tr>
-                      <th>Type</th>
-                      <th>Name</th>
-                  </tr>
-                </thead>
-                <tbody>
-                </tbody>
-            </table>
+          <ol id="<%= breadcrumb_id %>" class="breadcrumb breadcrumb-no-delimiter">
+          </ol>
+          <div class="spinner-border" role="status" id="loading-icon">
+            <span class="sr-only">Loading...</span> 
+          </div>
```

Comment:
I think this needs to be centered. I was able to do this by wrapping it.

![image](https://github.com/OSC/ondemand/assets/4874123/e98606c2-0add-47b2-acb3-a5f56002285e)

It should also be larger - though I can't do that as quickly.